test/spec/modules/nextrollBidAdapter_spec.js

```diff
@@ -0,0 +1,195 @@
+import { expect } from 'chai';
+import { spec, tryGetPubtag, hasCCPAConsent } from 'modules/nextrollBidAdapter';
+import * as utils from 'src/utils';
+
+const PUBTAG_LOCAL_STORAGE_KEY = 'nextroll_fast_bid';
+const PUBTAG_DATE_SUFFIX = '_set_date'
+
+describe('nextrollBidAdapter', function() {
+  let utilsMock;
+  beforeEach(function () {
+    // Remove to avoid side effects
+    localStorage.removeItem(PUBTAG_LOCAL_STORAGE_KEY);
```

Comment:
Where are you trying to set these ? I did not find any localstorage usage anywhere